rdkit/Chem/AtomPairs/Sheridan.py

```diff
@@ -13,26 +13,24 @@
 "Chemical Similarity Using Physiochemical Property Descriptors."
 J. Chem.Inf. Model. 36, 118-127 (1996)
 """
-from rdkit.DataStructs import IntSparseIntVect
+import os.path
+import re
+
 from rdkit import Chem
+from rdkit import RDConfig
 from rdkit.Chem import rdMolDescriptors
-from rdkit import DataStructs
-
 from rdkit.Chem.rdMolDescriptors import GetAtomPairFingerprint, GetTopologicalTorsionFingerprint
 
 numPathBits = rdMolDescriptors.AtomPairsParameters.numPathBits
 _maxPathLen = (1 << numPathBits) - 1
 numFpBits = numPathBits + 2 * rdMolDescriptors.AtomPairsParameters.codeSize
 fpLen = 1 << numFpBits
 
-import os.path, re
-from rdkit import RDConfig
-
 
 def _readPattyDefs(fname=os.path.join(RDConfig.RDDataDir, 'SmartsLib', 'patty_rules.txt')):
   with open(fname, 'r') as inf:
     lines = [x.strip().split('# ')[0].strip() for x in inf]
-  splitl = [re.split('[ ]*', x) for x in lines if x != '']
+  splitl = [re.split('[ ]+', x) for x in lines if x != '']
```

Comment:
This change addresses a warning that is now raised in Python 3.5 and will in future releases cause an exception. The old regex matches an empty string. Changing the * to + requires at least one space.